rebar.config

```diff
@@ -1,3 +1,8 @@
+{deps, [
+    {goldrush, ".*",
+        {git, "https://git-wip-us.apache.org/repos/asf/couchdb-goldrush.git",
+        {branch, "master"}}}
```

Comment:
Yeah, we removed rebar.config deps from subdependencies because it broke things, I don't think we can restore this.